### PR TITLE
San Antonio 

### PR DESCRIPTION
**Pull request by gvaldez7206**

### CODE REVIEW

File: src/main/java/frc/robot/commands/leds/MovingColorLEDs.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class MovingColorLEDs extends CommandBase {
     public MovingColorLEDs(LEDs leds, Color color, int count, boolean inverted) {
         this.leds = leds;
         this.color = color;
-        this.count = count;
+        this.count = count + 2;
```
Comment (wilsonwatson) [inline src/main/java/frc/robot/commands/leds/MovingColorLEDs.java:31]: Why are you adding it here? Why not when you construct it?

Comment (wilsonwatson) [reply src/main/java/frc/robot/commands/leds/MovingColorLEDs.java:31]: We'll fix this later.

File: src/main/java/frc/robot/commands/drive/TeleopSwerve.java [hunk 2]
```diff
@@ -47,9 +50,15 @@ public void execute() {
...
 
-        Translation2d translation =
-            new Translation2d(yaxis, xaxis).times(Constants.Swerve.MAX_SPEED);
-        double rotation = raxis * Constants.Swerve.MAX_ANGULAR_VELOCITY;
+        double angle_speed = Constants.Swerve.MAX_ANGULAR_VELOCITY;
+        double speed = Constants.Swerve.MAX_SPEED;
+        if (arm.getArmAngle() > -20) {
+            angle_speed /= 2;
+            speed *= 0.60;
+        }
```
Comment (legoguy1000) [inline src/main/java/frc/robot/commands/drive/TeleopSwerve.java:58]: I assume this worked well?

File: src/main/java/frc/robot/autos/MiddleScoreEngage.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import frc.robot.commands.drive.MoveToScore;
+import frc.robot.commands.wrist.WristIntakeRelease;
+import frc.robot.subsystems.Arm;
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.WristIntake;
+
+/**
+ * Score one game piece then engage from community side.
+ */
+public class MiddleScoreEngage extends SequentialCommandGroup {
```
Comment (legoguy1000) [inline src/main/java/frc/robot/autos/MiddleScoreEngage.java:21]: Was this actually tested? If not may want to move it to its own PR or at least remove it from the drop down so it's not accidentally run before testing

**Comment by aidan-underwood** [comment]
I'm not sure if this is ready to go. Will have to check tomorrow to make sure we don't have any outstanding changes on the laptop

File: src/main/java/frc/robot/autos/MiddleScoreEngage.java (hunk 3 above)
Comment (aidan-underwood) [reply src/main/java/frc/robot/autos/MiddleScoreEngage.java:21]: Was not tested. It should be moved to its own branch

File: src/main/java/frc/robot/commands/drive/TeleopSwerve.java (hunk 2 above)
Comment (wilsonwatson) [reply src/main/java/frc/robot/commands/drive/TeleopSwerve.java:58]: We tested it in the pits. I assume it worked out well. It was running throughout playoffs.

**Review by aidan-underwood — DISMISSED** [review]
ready to go

**Review by owenflatman — APPROVED** [review]
Looks good 👍

File: src/main/java/frc/robot/autos/MiddleScoreEngage.java (hunk 3 above)
Comment (owenflatman) [reply src/main/java/frc/robot/autos/MiddleScoreEngage.java]: Aidan did this.

**Review by owenflatman — APPROVED** [review]

**Review by wilsonwatson — APPROVED** [review]
